Componente de carrito

rafael1199v opened the pull request:
Instalación y configuración de Tailwind CSS para usarlo en el proyecto.

rafael1199v committed:
Instalación de Tailwind CSS

Instalación y configuración de Tailwind CSS para usarlo en el proyecto

rafael1199v committed:
Pagina de carrito

Implementación del diseño del sector del carrito sin funcionalidad.

RaidenMT commented on OrderNow-React/src/components/OrderCard.jsx:
Que los botones de - y + se muestren en cuadrado perfecto
![Screenshot_27-3-2025_172128_bookish-space-bassoon-wvvxvxxpv5jfwj6-5173 app github dev](https://github.com/user-attachments/assets/b0757775-7b95-467a-a633-65af896b448f)

rafael1199v replied:
Así se vería el carrito sin el contenido de los archivos index.css y App.css.
Se habló con el equipo DreamTeam requerían el contenido de esos archivos antes de eliminarlos y se usó el botón genérico.
La ruta para acceder al carrito es '/restaurant/:id/cart'
![Captura de pantalla 2025-03-28 225931](https://github.com/user-attachments/assets/e416fe5b-34c4-45d9-8521-1c1f775f5002)

RaidenMT commented on OrderNow-React/src/components/PayButton.jsx:
Que no sea un botón para pagar, sino un botón genérico y pasar texto

rafael1199v replied:
Ya se  modificó el componente para usar el botón genérico

RaidenMT commented:
Hacer pull antes de subir por favor

rafael1199v committed:
Merge remote-tracking branch 'upstream/main'

rafael1199v committed:
Eliminación de los archivos de estilo por default e integración del botón genérico

Se eliminó en contenido de los archivos index.css y App.css ya que no se van a utilizar en el proyecto y afectaban a la vista de algunos componentes.
Se integró el botón genérico para el carrito y se creó una ruta para acceder al componente del carrito para su visualización.

RaidenMT reviewed (approved):
Aprobado, en el siguiente sprint iremos afinando el front
